running-recipes/running-recipes.md

```diff
@@ -3,9 +3,8 @@
 * [Quickstart: Setting up your project and running recipes](/running-recipes/getting-started.md)
 * [Running Rewrite on a Gradle project without modifying the build](/running-recipes/running-rewrite-on-a-gradle-project-without-modifying-the-build.md)
 * [Running Rewrite on a Maven project without modifying the build](/running-recipes/running-rewrite-on-a-maven-project-without-modifying-the-build.md)
-* [Running Rewrite without build tool plugins](/running-recipes/running-rewrite-without-build-tool-plugins.md)
 * [Running Rewrite on a multi-module Maven project](/running-recipes/multi-module-maven.md)
-* [Popular recipe guides](/running-recipes/popular-recipe-guides/)
+* [Popular recipe guides](/running-recipes/popular-recipe-guides/README.md)
```

Comment:
Similar question here - was this link broken?

Reply:
Similarly: IDE warning on a broken link; figured the result is likely the same, but one less warning this way if it's all the same to you.